docs/mvvm/MigratingFromMvvmLight.md

```diff
@@ -0,0 +1,791 @@
+
+// Toolkit.Mvvm
+this.SetProperty(ref this.myProperty, value, true, nameof(this.MyProperty));
+```
+
+Note, string parameter is not required if the method is being called from the property's setter as it is inferred from the caller member name.
+
+#### Set<T> ( Expression, ref T, T, bool )
+
+`Set<T>(Expression, ref T, T, bool)` does not have a direct replacement. 
+
+It is recommended for improved performance that you replace this with the Toolkit's `SetProperty<T>(ref T, T, bool, string)` using the `nameof` keyword instead.
+
+```csharp
+// MvvmLight
+this.Set<MyObject>(() => this.MyProperty, ref this.myProperty, value, true);
+
+// Toolkit.Mvvm
+this.SetProperty<MyObject>(ref this.myProperty, value, true, nameof(this.MyProperty));
+```
+
+Note, string parameter is not required if the method is being called from the property's setter as it is inferred from the caller member name.
+
+#### Broadcast<T> ( T, T, string )
+
+`Broadcast<T>(T, T, string)` has a direct replacement which doesn't require a rename. 
+
+```csharp
+// MvvmLight
+this.Broadcast<MyObject>(this.oldValue, this.newValue, nameof(this.MyProperty));
+
+// Toolkit.Mvvm
+this.Broadcast<MyObject>(this.oldValue, this.newValue, nameof(this.MyProperty));
+```
+
+Note, the message sent via the `Messenger` property when calling the `Broadcast` method has a direct replacement for `PropertyChangedMessage` within the Toolkit's framework. 
+
+Also, string parameter is not required if the method is being called from the property's setter as it is inferred from the caller member name.
+
+#### RaisePropertyChanged<T> ( string, T, T, bool )
+
+There is no direct replacement for the `RaisePropertyChanged<T>(string, T, T, bool)` method and any code using this should be altered or removed.
+
+```csharp
+// MvvmLight
+this.RaisePropertyChanged<MyObject>(nameof(this.MyProperty), this.oldValue, this.newValue, true);
+
+// Toolkit.Mvvm
+// No direct replacement, remove
+```
+
+#### RaisePropertyChanged<T> ( Expression, T, T, bool )
+
+There is no direct replacement for the `RaisePropertyChanged<T>(Expression, T, T, bool)` method.
+
+It is recommended for improved performance that you replace this with the Toolkit's `RaisePropertyChanged<T>(string, T, T bool)` using the `nameof` keyword instead.
+
+```csharp
+// MvvmLight
+this.RaisePropertyChanged<MyObject>(() => this.MyProperty, this.oldValue, this.newValue, true);
+
+// Toolkit.Mvvm
+this.RaisePropertyChanged<MyObject>(nameof(this.MyProperty), this.oldValue, this.newValue, true);
+```
+
+### ViewModelBase Properties
+
+#### MessengerInstance
+
+`MessengerInstance` has a renamed direct replacement, `Messenger`.
+
+```csharp
+// MvvmLight
+IMessenger messenger = this.MessengerInstance;
+
+// Toolkit.Mvvm
+IMessenger messenger = this.Messenger;
+```
+
+#### IsInDesignMode
+
+There is no direct replacement for the `IsInDesignMode` property and any code using this should be altered or removed.
```

Comment:
We might want to mention that this (and `IsInDesignModeStatic` too) have been removed on purpose because they required platform-specific support and could therefore not be part of the same portable .NET Standard set of helper types that we wanted the MVVM Toolkit to be made up of? As in, I'd argue that having a property such as this directly within the base `INotifyPropertyChanged` type breaks code modularity a bit, as it forces the type to have strong dependencies to either platform-specific APIs, or to additional service interfaces to abstract them Conceptually, neither of these two should belong to this type.

Reply:
Done 👍